downstairs/src/region.rs

```diff
@@ -639,17 +577,7 @@ impl Region {
             "eid:{} Found repair files: {:?}", eid, repair_files
         );
 
-        // Depending on if this is a clone or not, we have a different
-        // set of files we expect to find.
```

Comment:
What about different backends in the future? I guess we would have to imagine a new multi-file backend to argue for some version of this code to stick around, so maybe this is ok.

Reply:
Yeah, we can always bring multi-file support back, but I don't have any plans for multiple-file-per-extent backends...